airdrop/api/authorization.ts

```diff
@@ -7,6 +7,7 @@ export default async function (req: VercelRequest, res: VercelResponse) {
   const auth = qs.parse(req.query);
   const accessToken = auth.access_token;
   res.setHeader('Set-Cookie', `x-access-token=${accessToken};httpOnly;secure;path=/;`);
-  res.setHeader('Location', '/crab-smart-chain/get-started/darwinia-crab?oauth=github#get-tokens');
-  res.end('Redirect /crab-smart-chain/get-started/darwinia-crab?oauth=github#get-tokens');
+  res.setHeader('Location', '/evm-compatible-crab-smart-chain/get-started/darwinia-crab?oauth=github#get-tokens');
```

Comment:
I am not so sure whether this url update would lead to any problem or not, It is better to let other guys who are familiar with airdrop take a review about this.

Reply:
Others changes. LGTM

Reply:
@fewensa 帮忙看下这里有问题不